templates/Layout/PartialUserForm.ss

```diff
@@ -0,0 +1,17 @@
+<div class="container">
+    <% if $Breadcrumbs %>
+        <% include Breadcrumbs Breadcrumbs=$Breadcrumbs %>
+    <% end_if %>
+    <h1>$Title</h1>
+
+    <% if $Link %>
+        <p>
+            This form allows for multiple sessions (singular or multiple users).
+            You can access or contribute to this form using the following link:
+            <a href="$Link">$Link</a>
+        </p>
+    <% end_if %>
+
+    $Content.RichLinks
```

Comment:
RichLinks is CWP specific, could you use just `Content`  and make a MBIE copy with `.RichLinks`

Reply:
Will remove, copied it from mbie's UserDefinedForm template